go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/spf13/cast v1.6.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spiffe/go-spiffe/v2 v2.1.6
-	github.com/stretchr/testify v1.8.4
+	github.com/stretchr/testify v1.9.0
```

Comment:
I know this can be a problem. There is one piece of code in our functional test framework that uses a deprecated method that returns Fatal at 1.9.x - if this is not required, please, keep this with an old version via a replace statement.

Reply:
I can fix that no problem :)